--- elisa-0.3.5/elisa/core/plugin_registry.py.old	2008-03-06 17:56:59.000000000 +0100
+++ elisa-0.3.5/elisa/core/plugin_registry.py	2008-05-03 01:05:26.000000000 +0200
@@ -392,7 +392,7 @@
                     self.warning("Error loading plugin: " + str(exc))
                 continue
             
-            full_path = inspect.getsourcefile(py_object)
+            full_path = inspect.getfile(py_object)
             plugin_dir = os.path.dirname(full_path)
 
             distribution_file = ""
